src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  display: flex;
+  font-weight: 500;
+  font-size: 12px;
+  text-align: center;
+  color: #000;
+  text-transform: uppercase;
+  height: 60px;
+  align-items: center;
+}
+
+.nav__item:not(:first-child) {
+  padding-left: 20px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:active {
+  color: #00acdc;
+  box-shadow: 0 4px 0 #00acdc;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  background-color: #00acdc;
```

Comment:
Use a CSS variable for the color of the pseudo-element for the active link to maintain consistency.